docs/databags.md

```diff
@@ -109,6 +109,7 @@ An apps hash is a contextual (see above) Hash, part of a plaintext data bag item
 * `[app]['download']['version']` - Expected [version number][app.conf] (if any) used to determine if a new app should be downloaded.
 * `[app]['files']` - Hash of files to manage under the "default" or "local" directory.
 * `[app]['files'][filename]` - Contents of a particular file to manage. It can take 3 values, a hash of stanzas -> key-value pairs (then written with the splunk template), a string (written as is), or nil / false (deleted). If the hash or string is empty, the file is also deleted.
+* `[app]['lookups']` - Hash of lookup files for the app. The key in the hash will be the name of file when it lands on splunk and value will be the url to lookup file. To delete an existing lookup file, set the value of the lookup file to `false` or `null` or to an empty string. Only supported file name formats are .csv, .csv.gz and .kmz. If you are downloading a comma separated text file (.txt), the file name you specify as the key in the hash should have a .csv extension. Please see [splunk docs](http://docs.splunk.com/Documentation/Splunk/6.3.2/Knowledge/Addfieldsfromexternaldatasources) for more information about the supported file formats. `[app]['files']` hash can be used to specify any .conf setting that is required for the lookup.
```

Comment:
`when it lands in the Splunk app` ?

`The only supported file name extensions are` ? (if so then the sentence afterward about text file is redundant).